test/exercism/comment_test.rb

```diff
@@ -26,7 +26,7 @@ def test_mentions
       ["Mention @alice and @bob and also @charlie", %w(alice bob), "ignore unknown users"],
     ].each do |body, mentions, _desc|
       usernames = User.where(id: Comment.new(body: body).mention_ids).map(&:username)
-      assert_equal usernames, mentions
+      assert_equal usernames.sort, mentions.sort
```

Comment:
This is not your bug, but the arguments here are the wrong way around, it should be:
`assert_equal expected, actual`

The `sort`s are a good solution to the reported non-deterministic ordering issue.